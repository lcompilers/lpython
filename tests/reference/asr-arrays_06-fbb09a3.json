{
    "basename": "asr-arrays_06-fbb09a3",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/arrays_06.py",
    "infile_hash": "670cbcac5e942bae5293cd94090bab65a2039384d71fb8be4c6db5a1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-arrays_06-fbb09a3.stderr",
    "stderr_hash": "1fa3f5061a72f03c0678806c0460b9ec5caf01cbbd2f07a606f1057e",
    "returncode": 2
}