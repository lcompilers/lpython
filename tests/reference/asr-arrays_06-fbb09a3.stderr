semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> tests/errors/arrays_06.py:6:5
  |
6 |     x: i16[5, 4] = empty([5, 4], dtype=int32)
  |     ^              ^^^^^^^^^^^^^^^^^^^^^^^^^^ type mismatch ('i16[5,4]' and 'i32[5,4]')
